modules/retail/pages/retail-uyuni-branchserver.adoc

```diff
@@ -0,0 +1,146 @@
+// Feedback provided by Lukas
+== Create Activation Key for a Branch Server and the Retail Terminals
+
+The branch server is based on the {productname} Proxy.
+Its activation key must contain these child channels:
+
+* openSUSE Leap {opensuse-version} Updates (x86_64)
+* Uyuni Client Tools for openSUSE Leap {opensuse-version} (x86_64)
+* Uyuni Proxy Stable for openSUSE Leap {opensuse-version} (x86_64)
+
+The activation key for retail terminals based on openSUSE Leap {opensuse-version} must contain these child channels:
+
+* openSUSE Leap {opensuse-version} Updates (x86_64)
+* Uyuni Client Tools for openSUSE Leap {opensuse-version} (x86_64)
+
+For more information about creating activation keys, see xref:client-configuration:activation-keys.adoc[].
+
+
+
+== Register the Branch Server and Terminals as Clients
+
+You register both the branch server and the terminals as openSUSE clients.
+To register your openSUSE clients, you need a bootstrap repository.
+By default, bootstrap repositories are automatically created, and regenerated daily for all synchronized products.
+You can manually create the bootstrap repository from the command prompt, using this command:
+
+----
+mgr-create-bootstrap-repo --with-custom-channels
+----
+
+For more information on registering your clients, see xref:client-configuration:registration-overview.adoc[].
+
+
+
+=== Register the Branch Server
+
+A retail branch server is registered as an openSUSE proxy.
+The proxy can be bootstrapped using the {webui}, or at the command prompt.
+Ensure you use the activation key you created for the proxy.
+
+For more information about proxies, see xref:installation:uyuni-proxy-registration.adoc[].
+For more information about activation keys, see xref:client-configuration:activation-keys.adoc[].
+
+
+
+.Procedure: Setting Up the Uyuni Proxy
+. Check  that the ``Uyuni Proxy Stable for openSUSE Leap {opensuse-version} (x86_64)`` channel is assigned to the proxy on the system profile page.
+
+. At the command prompt on the proxy, as root, install the proxy pattern:
++
+
+----
+zypper in -t pattern uyuni_proxy
+----
+
+. Finalize the proxy setup:
++
+----
+configure-proxy.sh
+----
++
+  [command]``configure-proxy.sh`` is an interactive script.
+  For more information about the proxy setup script, see xref:installation:uyuni-proxy-setup.adoc#uyuni-proxy-setup-confproxy[].
+
+. OPTIONAL: If you want to use the same system also as a build host, navigate to the client's system profile and check [systemitem]``OS Image Build Host`` as a [guimenu]``Add-On System Types``.
+
+. Configure the proxy to run as a branch server.
+  For example:
++
+----
+retail_branch_init <branch_server_minion_id> --dedicated-nic eth1 \
+    --branch-ip 192.168.7.5 \
+    --netmask 255.255.255.0 \
+    --dyn-range 192.168.7.100 192.168.7.200 \
+    --server-domain branch.example.org \
+    --branch-prefix uyuni
+----
++
+  For additional options,  use the [command]``retail_branch_init --help`` command.
```

Comment:
This is run on Uyuni Server. So start with something "On Uyuni server run `retail_branch_init ...`"